svg/intro/script-content.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Test: </title>
+<link rel="help" href="https://svgwg.org/svg2-draft/intro.html#AboutSVG">
```

Comment:
OK, I will also add all the other SVG elements to check that script is valid inside all of them. I won't add dozens of help links.

Reply:
Yeah, probably a help link to the script definition is enough.

I didn't go as far as recommending including _every_ SVG element, because I think you've got a pretty good cross-section of the different types of elements (shapes, effects, groups, etc.). But then again: you never know which one might have been overlooked in an implementation.

Reply:
Agree with @AmeliaBR that the rel=help should point to https://svgwg.org/svg2-draft/interact.html#ScriptElement